aws-sagemaker-hosted-scorer-cpp/py/scorer/mojo_cpp_scorer.py

```diff
@@ -0,0 +1,124 @@
+from flask import Flask, request
+from flask_restful import Resource, Api
+import logging
+import json
+import os
+import threading
+import daimojo.model
+import datatable as dt
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+app = Flask(__name__)
+api = Api(app)
+
+
+class MojoPipeline(object):
+    _model = None
+    _instance = None
+    _lock = threading.Lock()
```

Comment:
Why do you need this if you don't reuse `MojoPipline`?

Reply:
As this is the initial version, implemented like this because, modifications might be add depends on feedbacks. Hence I thought it could be reuse.

Reply:
It's better to add it when you use it. For not it's just an extra code and complexity that adds nothing. Plus we need to test mojo more carefully for reuse cases.

Reply:
This is fine, thread-safe singleton in case Flask is running with multi-threading.